packages/lab/src/Blade/Blade.styles.tsx

```diff
@@ -0,0 +1,89 @@
+import { createClasses, outlineStyles } from "@hitachivantara/uikit-react-core";
+import { theme } from "@hitachivantara/uikit-styles";
+
+export const { staticClasses, useClasses } = createClasses("HvBlade", {
+  root: {
+    position: "relative",
+
+    display: "flex",
+    minWidth: 72,
+
+    "& + $root": {
+      marginLeft: theme.spacing("sm"),
+    },
+
+    transition: "flex-grow 800ms linear",
```

Comment:
The problem with ease is that there are multiple transitions occurring: at least two (flex-grow on the blade, max-width on the content), possibly more (while one blade opens, others close / shrink). Linear makes it easier to avoid strange bumps (although they still happen sometimes).

I did have it (ease-in opening, ease-out closing)... But that stupid max-width hack really messes things. Please be free to try out different options, maybe you're luckier than me.

Reply:
I did not notice the strange bumps on my side 🤔

Let's keep it like this then; as it's only styling, we can always improve on later anytime 👍

Reply:
I'll keep the linear and speed up to 600ms.